app/src/main/java/ch/epfl/qedit/view/home/HomeQuizListFragment.java

```diff
@@ -41,14 +43,15 @@
     private Handler handler;
 
     private ProgressBar progressBar;
-    private ListEditView.Adapter<Map.Entry<String, String>> listAdapter;
+
+    private ListEditView.Adapter<Map.Entry<String, String>, SearchableMapEntry> listAdapter;
 
     private ConfirmDialog deleteDialog;
     private EditTextDialog addDialog;
     private int deleteIndex;
 
     private User user;
-    private List<Map.Entry<String, String>> quizzes;
+    private SearchableMapEntry quizzes = new SearchableMapEntry();
```

Comment:
This is a list right? So there are several searchable quizzes but the `SearchableMapEntry` suggests only one entry which is confusing to me.